src/utils/setupOutputFileSystem.js

```diff
@@ -17,27 +15,9 @@ function setupOutputFileSystem(context) {
   if (context.options.outputFileSystem) {
     const { outputFileSystem: outputFileSystemFromOptions } = context.options;
 
-    // Todo remove when we drop webpack@4 support
-    if (typeof outputFileSystemFromOptions.join !== "function") {
-      throw new Error(
-        "Invalid options: options.outputFileSystem.join() method is expected"
-      );
-    }
-
-    // Todo remove when we drop webpack@4 support
-    // @ts-ignore
-    if (typeof outputFileSystemFromOptions.mkdirp !== "function") {
-      throw new Error(
-        "Invalid options: options.outputFileSystem.mkdirp() method is expected"
-      );
-    }
```

Comment:
@alexander-akait Do we need this with webpack v5?

Reply:
Otherwise, I'll remove test cases for this as well, which are currently failing.

Reply:
If this tests for webpack  - Yes

Reply:
We have a lot of integration tests, so we will catch if it will have problems